public/video-ui/styles/layout/_topbar.scss

```diff
@@ -3,6 +3,16 @@ $topbarHeight: 50px;
 .topbar {
   background: $color700Grey;
   border-bottom: 1px solid $greyBorderColor;
+
+  &--training-mode {
+    background: repeating-linear-gradient(
+        -45deg,
+        $color500Grey,
+        $color500Grey 10px,
+        $color800Grey 10px,
+        $color800Grey 20px
```

Comment:
Would adding opacity here make the title a little bit more visible when in training mode?